examples/spatial_indexing_1d_grids.py

```diff
@@ -0,0 +1,103 @@
+"""
+Spatial indexing of 1D grids
```

Comment:
I'd state:

"Spatial indexing of 1D networks and linear geometry"

Reply:
Done!